src/lib.rs

```diff
@@ -2257,3 +2312,107 @@ pub mod roaring {
         _inner: Results<[u64; 1024], Vec<u16>>,
     }
 }
+
+pub use chain_mod::{chain, chain_one};
+
+pub mod chain_mod {
+
```

Comment:
If you could put a comment in here explaining what is going on (yes it is the same as `iter::chain`, except that we can put `inline(always)` on the `next` method).